sklearn/metrics/base.py

```diff
@@ -20,10 +20,6 @@
 from ..utils.multiclass import type_of_target
 
 
-class UndefinedMetricWarning(UserWarning):
```

Comment:
Only way to deprecate importing `UndefinedMetricWarning` is to do 

```
from ..exceptions import UndefinedMetricWarning
UndefinedMetricWarning = deprecated("msg")(UndefinedMetricWarning)
```

But that will warn only upon `__init__`, is that okay?